docs/src/pages/demos/selects/SimpleSelect.js

```diff
@@ -218,9 +224,7 @@ class SimpleSelect extends React.Component {
         </FormControl>
         <FormControl variant="outlined" className={classes.formControl}>
           <InputLabel
-            ref={ref => {
-              this.labelRef = ReactDOM.findDOMNode(ref);
```

Comment:
`findOMNode` is not actually used inside the render. It's used inside the callbackref.

Reply:
Ah ok, my solution is dirty. Tried it with the https://material-ui.com/api/root-ref/ but still needed a re-render to fix first value.

You only have the offsetwidth after mounting the component.  So it could be the old code but it needs to re-render the component after having the reference.

@oliviertassinari I think there should indeed be some context of rendering the input label first and after that the input or something.